lib/ruby_warrior/abilities/challenge.rb

```diff
@@ -0,0 +1,37 @@
+module RubyWarrior
+  module Abilities
+    class Challenge < Base
+
+      def description
+        "Initiates a challenge between a queen bug and the warrior."
+      end
+
+      def perform(direction = :forward, question, acceptable_answers)
+        verify_direction(direction)
+        receiver = unit(direction)
+        if receiver
```

Comment:
~~Should we check to ensure that `receiver` is the queen? Otherwise it seems like theoretically this ability could be used to insta-kill any unit~~

My bad, realized this ability is given to the queen, not the warrior. Had it the wrong way around in my head